src/main/kotlin/org/ageseries/libage/sim/Material.kt

```diff
@@ -1,90 +1,37 @@
-		override val thermalConductivity: Double = 79.5
-        override val specficHeat: Double = 0.0
-        override val density: Double = 7874.0
-	};
-
-    // Density of the material (km/m^3)
-    abstract val density: Double
-
-/*
-Density (STP)
-
-H2O 	    1,000
-Iron	    7,874
-Copper	    8,950
-Tungsten	19,250
-Gold	    19,300
-Platinum	21,450
-
-Source: https://www.calculator.net/density-calculator.html
- */
-
-	// resistivity (ohms/meter)
-	abstract val resistivity: Double
-/*
-Electrical Resistivity
-Material    Resistivity ρ (ohm m)
-Silver:     1.59    x10^-8
-Copper:     1.68    x10^-8
-Aluminum:   2.65    x10^-8
-Tungsten:   5.6     x10^-8
-Iron:       9.71    x10^-8
-Platinum    10.6    x10^-8
-Lead:       22      x10^-8
-Mercury     98      x10^-8
-Glass       1-10000 x10^9
-Rubber      1-100   x10^13
-Source: http://hyperphysics.phy-astr.gsu.edu/hbase/Tables/rstiv.html
-*/
-
-	// thermal conductivity (W/m K)
-	abstract val thermalConductivity: Double
-
-/*
-Thermal Conductivity
-Material    Thermal conductivity (W/m K)*
-Diamond:    1000
-Silver:     406.0
-Copper:     385.0
-Gold:       314
-Brass:      109.0
-Aluminum:   205.0
-Iron:       79.5
-Steel:      50.2
-Lead:       34.7
-Mercury:    8.3
-Glass:      0.8
-Water:      0.6
-Air:        0.024
-Source: http://hyperphysics.phy-astr.gsu.edu/hbase/Tables/thrcn.html
-*/
-
-    abstract val specficHeat: Double
-
-	// amperage per material type per mm^2 (possibly - calculate from thermal conductivity, power and thermal conductivity
-	// abstract val amperageMaterial: Double
-}
+class Material(
+	/** Intrinsic linear electrical resistance, in Ohm/m. */
+	val electricalResisitivity: Double,
+	/** Intrinsic linear thermal conductivity (on one axis), in W/mK. */
+	val thermalConductivity: Double,
+	/** Intrinsic specific heat capacity, in J/Kkg. */
+	val specificHeat: Double,
+	/** Density at STP, in g/cm^3 (or kd/dm^3). */
+	val density: Double,
+) {
+	override fun toString() =
+		"<Mat ${electricalResisitivity}Ohm/m ${thermalConductivity}W/mK ${specificHeat}J/Kkg ${density}g/cm^3>"
+
+	companion object {
+		// Good sources for this information:
+		// Electrical resistivity: https://en.wikipedia.org/wiki/Electrical_resistivity_and_conductivity
+		// Thermal conductivity: https://en.wikipedia.org/wiki/List_of_thermal_conductivities
+		// Specific heat: https://en.wikipedia.org/wiki/Table_of_specific_heat_capacities
+		// 		(use STANDARD_TEMPERATURE; note that the table has J/Kg, not J/Kkg!)
+		// 		alternatively: https://www.engineeringtoolbox.com/specific-heat-solids-d_154.html
+		//		(also uses kJ/Kkg = J/Kg!)
+		// Density: https://en.wikipedia.org/wiki/Density
+		val COPPER = Material(1.68e-8, 401.0, 385.0, 8.94)
+		val LATEX_RUBBER = Material(1e13, 0.134, 1130.0, 0.92)
+		val IRON = Material(9.7e-8, 83.5, 412.0, 7.87)
+		val GLASS = Material(1e13, 1.11, 840.0, 2.5)
+		val ALUMINUM = Material(2.65e-8, 100.0, 897.0, 2.7)
+		val ALUMINIUM = ALUMINUM  // for our UK friends
```

Comment:
This has the potential to cause issues with serialization. I think it's unnecessary complexity to support different English dialects in the codebase.